modules/ocf_printhost/files/enforcer

```diff
@@ -35,6 +37,21 @@ CONF_FILE = '/opt/share/enforcer/enforcer.conf'
 
 Message = namedtuple('Message', ['subject', 'body'])
 
+# Print Job Statuses:
+# 0 - Completed
+# 1 - Pending
+# 2 - Error (problematic file)
+# 3 - Error (insufficient quota)
+# 4 - Error (job processing)
+
```

Comment:
nit: extra lines

Reply:
Autopep8 fails when removing this newline.

Reply:
I see, it seems PEP8 wants block comments separated by two lines. Maybe either inline the comments next to the line itself? Though I don't mind.

Reply:
Consider using inline comments and a doc string?
```python
class PrintJob(Enum):
  """A print job status."""
  COMPLETED = 0
  PENDING = 1
  # Error: problematic file
  FILE_ERROR = 2
  # Error: insufficient quota
  QUOTA_LIMIT = 3
  # Error: job processing
  JOB_ERROR = 4